extensions/mongoProcessor/MongoQueueProcessor.js

```diff
@@ -151,6 +160,41 @@ class MongoQueueProcessor {
         return this._consumer.close(done);
     }
 
+    _isMasterKey(entry) {
+        return entry.getObjectVersionedKey() === entry.getObjectKey();
```

Comment:
Looks like this comparison is already exposed by `ObjectQueueEntry` as the method `isVersion`. We could avoid some duplicate code by using that. https://github.com/scality/backbeat/blob/93980e8130a0a7c25d9f0ec52234e6b5d46e49ab/lib/models/ObjectQueueEntry.js#L77

Reply:
For this, I found a method in arsenal for checking master key. I'll use that because I think it more accurately performs the check I need and also works for readability